meetings/2019-10-29.md

```diff
@@ -0,0 +1,95 @@
+- early Doodle and clear announcement helps for attendance
+- the work on the guide is done via PRs (merge whenever it makes sense)
+
+## Decide on workshop
+
+### Which audience
+
+- start with intermediate:
+  - works best for first workshop
+  - quick vote: we start with intermediate
+- "intermediate" means confident with with computers
+  - this of course includes juniors
+
+### Which content
+
+- recap:
+  - nerves project (we stopped looking at it)
+  - snake (people seem to like it)
+  - elixir-girls chat app (only looked at it briefly)
+    - we did not look into their intermediate guide
+- round of votes:
+  - snake fun and entertaining
+  - snake lots of Elixir good to start
+  - snake has visual aspects
+  - snake tutorial has "natural steps" (works great for guide)
+  - snake does not work for finding a job, but valid for tutorial
+  - hardware project: makes sense for the future, but too much orga effort for now
+  - elixir girls app: we would need to update the guide (no ETS, less JS)
+    - pro: great for lots of elixir and no phoenix
+    - pro: has lots of the language features in it
+- decision: take the snake tutorial
+- will proceed with other projects in future workshops then
+
+#### What else do we need for the workshop?
+
+- we have simplified the original tutorial
+- split the tutorial into steps
+- brief introduction into the language:
+  - best as an interactive presentation (it should fit what we need for the game)
+  - also a guide for what is what
+- do we need the repo for people to clone it?
+  - not necessary... but good as a solution
+  - we'd prefer to have people start with an empty file
+  - have repo with steps as a solution
+- break the guide down into steps
+  - every step relates to a visual change in the game
+  - have code snippets and where to put them in the guide
+  - code snippets are explained via guide plus coaches
+  - include information for coaches in the guide (for people not familiar with scenic)
+  - guide for participants and coaches in the same file (e.g. use HTML detail tag)
+- start from the `scenic_new` scaffold
+  - small steps
+  - bigger chapters
+  - start with Markdown on Github (can then also maybe be hosted on new Code Curious website)
+- have group working for the guide
+  - flash out basic chapter structure and one example chapter together
+  - then work on different parts in parallel
+
+### Which date
+
+- beginning of February sounds like a plan
+  - early February (Saturday 1st of February)
+- we need 6 weeks for preparation
+
+## Outcome
+
+- **Nicolo** will set up Doodle for next meetup
+- do an intermediate workshop with the snake tutorial
+  - write tutorial for attendees setting up snake game from scratch (using `scenic_new`)
+  - **Max, Nico and Markus** meet on Monday 4th to start with the guide (flash out structure, write first chapter)
+- **Tobi** is doing a general intro guide to Elixir (what's necessary for the snake tutorial)
```

Comment:
I didn't explicitly say it, but unless someone needs it earlier I'd likely do this in December as I probably won't have a project in december aka lots of time. I might start earlier though :)

Reply:
Added a note for that.